src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -1129,12 +1129,20 @@ void LottieBuilder::updateMaskings(LottieLayer* layer, float frameNo)
     auto pMask = static_cast<LottieMask*>(layer->masks[0]);
     auto pMethod = pMask->method;
     auto opacity = pMask->opacity(frameNo);
+    auto expand = pMask->expand(frameNo);
 
     auto pShape = layer->pooling();
     pShape->reset();
     pShape->fill(255, 255, 255, opacity);
     pShape->transform(layer->cache.matrix);
-    pMask->pathset(frameNo, P(pShape)->rs.path.cmds, P(pShape)->rs.path.pts, nullptr, nullptr, nullptr, exps);
+
+    if (expand == 0.0f) pMask->pathset(frameNo, P(pShape)->rs.path.cmds, P(pShape)->rs.path.pts, nullptr, nullptr, nullptr, exps);
+    else {
+        //TODO: Once path direction support is implemented, ensure that the direction is ignored here
+        auto offset = LottieOffsetModifier(pMask->expand(frameNo));
+        pMask->pathset(frameNo, P(pShape)->rs.path.cmds, P(pShape)->rs.path.pts, nullptr, nullptr, &offset, exps);
```

Comment:
Suggestion:  `(expand > 0) ? LottieOffsetModifier(pMask->expand(frameNo)) : nullptr`

Reply:
expand can be < 0 as well. and I can not take address of a temp variable. please check now

Reply:
sorry, reverted. I can use new or unique. do you think it will be better? I didn;t want to allocate on heap

Reply:
nevermind, it's already pretty good. thanks.